### ISSUE TITLE
reduce produced binary size

### ISSUE THREAD

**Issue by Anton-4** [roc-lang/basic-cli#107]
Roc apps built with basic-cli can have a size in excess of 100MB, this should be investigated and reduced.

**Comment by mikevdg** [comment]
Hello World with --opt-size and stripped is  2346704 bytes.

**Comment by Anton-4** [comment]
Yeah, we improved this and I forgot about this issue, thanks for alerting me @mikevdg!
